docs/INSTALLATION.md

```diff
@@ -0,0 +1,11 @@
+# Windows Installation
+
+Powershell is included in most versions of windows
```

Comment:
Windows needs to be capitalized, I think (Microsoft capitalizes it everywhere).

Also, the sentence needs a period.